### DISCUSSION TITLE
snow2 errors with version 2.1

### DISCUSSION THREAD

**Discussion by nuritg** [PMEAL/porespy#707] [Q&A]
Hello,
When I try to run this example (https://porespy.org/examples/networks/tutorials/snow_basic.html)
I get the following error after running snow2:
TqdmKeyError: "Unknown argument(s): {'colour': None}"
I'm running porespy version 2.1
Any help would be most welcome :)
Thanks

**Comment by jgostick** [comment]
It seems like your ``tqdm`` is different than the one we used to make the example, and that it is complaining about the 'color' of the progress bar.  Try doing this:

``` python
import porespy as ps
ps.settings.tqdm = {}  # Just set all the settings to the default values

```

FYI, you can see all the setting in porespy using:

``` python
print(ps.settings)
```

**Reply by nuritg** [reply]
It works! Thank you :)
